themes/SuiteR/include/DetailView/DetailView.tpl

```diff
@@ -0,0 +1,240 @@
+{*
+/*********************************************************************************
```

Comment:
[License](https://suitecrm.com/wiki/index.php/Coding_Standards) header should be updated.